README.md

```diff
@@ -700,6 +701,25 @@ interface Table extends Parent {
 
 **Table** represents 2d data.
 
+### CustomCodeComponent
+
+```ts
+interface CustomCodeComponent extends Parent {
+  type: "custom-code-component"
+  path: string
+  versionRange: string
+  lastModified: string
```

Comment:
The `lastModified` field is already returned from CAPI for all content types we support. It is used every time there is a republish of the content in the platform. I suppose the meaning here is quite different so can please rename it. As CCC is going as a separate content type there will be mismatch of what lastModified means for CCC and all of the other content types.

Reply:
have changed this 👍